lte/gateway/deploy/roles/ovs_build/vars/all.yaml

```diff
@@ -1,14 +1,2 @@
-# ovs version on github default v2.8.1
-ovs_version: "v2.8.1"
-# ovs version short in order to retreive patches if any
-ovs_version_short: "2.8.1"
-# patches relative location
-gtp_patch_path: "third_party/gtp_ovs"
```

Comment:
Why did all of these get deleted with no change to the actual role? Did I miss a prior change?

Reply:
Those are unused in the role. It was just a dumb copy paste